jupyter-notebooks/data-api-tutorials/clip_and_ship_introduction.ipynb

```diff
@@ -0,0 +1,194 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Item Clipping and Download\n",
+    "\n",
+    "This example clips, downloads and unzips a given scene to the AOI provided, returning only the image data contained in with the AOI. To so this, we use the \"Clips\" API.\n",
+    "\n",
+    "In this example, we are clipping a scene to contain only Golden Gate Park in San Francisco."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Input Parameters "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from os import environ\n",
+    "\n",
+    "# Set API key (this should to be an environment variable)\n",
+    "api_key = environ['PL_API_KEY']\n",
```

Comment:
I just found this: PLANET_API_KEY = os.getenv('PL_API_KEY'). But that is like that by design I think.